notify/impl.go

```diff
@@ -753,10 +753,16 @@ func (n *OpsGenie) Notify(ctx context.Context, as ...*types.Alert) (bool, error)
 		apiURL = n.conf.APIHost + "v1/json/alert/close"
 		msg = &opsGenieCloseMessage{&apiMsg}
 	default:
+		message := tmpl(n.conf.Message)
+		if len(message) > 130 {
+			message = message[:130]
```

Comment:
From an end-user standpoint, I think doing an ellipsis would be a good clue to the alertreceiver that something is being truncated, so they can update their config message. Something like

```go
if len(message) > 130 {
    message = message[:127]+"..."
    log.With...
}
```

Thoughts?

I think this is being done somewhere else but I'm not sure..
@brancz @brian-brazil

Reply:
In 2 places we just truncate, in 1 place we add an ellipsis.

Reply:
I like the ellipsis as well. Should we change all other msgs to be consistent using it?

Reply:
Let's do the ellipsis here, and if it makes sense in the other two places, move those to ellipses as well.

Reply:
ellipsis is appropriate I think

Reply:
(the ellipses changes for the other places should be in a different pr, if there was any confusion)